Tecchly/DLOplanTool
# Dlo 11

### **Related Issue/Keyword:**
[DLO-11](https://trello.com/c/0rOVuimP/11-guidance-and-instruction)
### **Description:**
there is a lot of problems for this guidance if not followed by steps, but for demo purpose I think it's so-so
### **Testing:**

**Steps for manual testing:**

1. delete your user document in firestore or login with a new google account
2. the guidance will be shown automatically (or if you skip the first step, then manually click the question mark icon on the right top of the home page)
3. follow the guidance steps to create a new project, if the step focus on an interactive element on the page( such as a button or an input field), then should click the button or input something and go next step, except for the last steps after going back to home page from mapping page

### **Checklist:**
